src/pages/Shop/ShopProduct/ShopProduct.js

```diff
@@ -0,0 +1,166 @@
+import React, { useState, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import styled from 'styled-components';
+
+const ShopProduct = () => {
+  const [shopProductData, setShopProductData] = useState([]);
+  const [sortFilter, setSortFilter] = useState('옵션 선택');
+
+  const navigate = useNavigate();
+
+  const goToDetail = () => {
+    navigate('/detail');
+  };
+
+  const onChangeSortTitle = e => {
+    setSortFilter(e.target.value);
+  };
+
+  useEffect(() => {
+    fetch('/data/productData.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data => {
+        setShopProductData(data);
+      });
+  }, [sortFilter]);
+
+  return (
+    <ShopProductWrapper>
+      <ProductSortSelectBox>
+        <ProductSortSelect onChange={onChangeSortTitle}>
+          {SHOP_SELECT_SORT.map(({ id, value, title }) => (
+            <option key={id} value={value}>
+              {title}
+            </option>
+          ))}
+        </ProductSortSelect>
+      </ProductSortSelectBox>
+      <ShopProductList>
+        {shopProductData.map(product => {
+          const productPrice = product.price.toLocaleString();
+
```

Comment:
how about
```suggestion
        {shopProductData.map(product => {
          const { id, thumnailImageUrl, enName, krName, brandName, price : _price } = product;
          const price = _price.toLocaleString();
          
```

이렇게 한 줄을 추가함으로써 바인딩 하는 쪽에서 편하게 사용 할 수 있을 것 같습니다!

Reply:
넵 감사합니다 ! 수정 완료했습니다 :)